### PR TITLE
add referring provider to FhirEncounterService

### LINKED ISSUE #4866: referring provider for FHIR
### Describe the problem
Once we add a referring provider to the encounter from #4860 we will want to add this to the FHIR interface as a participant with a participant type of REF.

### PR DESCRIPTION
**Pull request by stephenwaite**
Fixes #4866

### CODE REVIEW

File: src/Services/FHIR/FhirEncounterService.php [hunk 1]
```diff
@@ -1,5 +1,23 @@
...
+ * @author    Yash Bothra <yashrajbothra786@gmail.com>
+ * @author    Stephen Waite <stephen.waite@cmsvt.com>
+ * @author    Vishnu Yarmaneni <vardhanvishnu@gmail.com>
+ * @author    Brady Miller <brady.g.miller@gmail.com>
+ * @author    Stephen Nielson snielson@discoverandchange.com
+ * @copyright Copyright (c) 2020 Yash Bothra <yashrajbothra786@gmail.com>
+ * @copyright Copyright (c) 2020, 2022 Stephen Waite <stephen.waite@cmsvt.com>
+ * @copyright Copyright (c) 2020 Vishnu Yarmaneni <vardhanvishnu@gmail.com>
+ * @copyright Copyright (c) 2021 Brady Miller <brady.g.miller@gmail.com>
+ * @copyright Copyright (c) 2022 Stephen Nielson <stephen@discoverandchange.com>
```
Comment (adunsulag) [inline src/Services/FHIR/FhirEncounterService.php:17]: Its snielson@discoverandchange.com

**Review by adunsulag — APPROVED** [review]
Everything looked fine to me, noticed one minor issue with my email lol, but everything else looked fine from the API standpoint.

**Comment by adunsulag** [comment]
Looks good @stephenwaite, feel free to bring this in, or I can bring it in.

**Comment by stephenwaite** [comment]
thanks for reviewing, bringing in

**Comment by bradymiller** [comment]
2DV

**Pull request merged**